lib/config/options/index.ts

```diff
@@ -2088,6 +2090,8 @@ const options: RenovateOptions[] = [
     type: 'string',
     default:
       '{{{commitMessagePrefix}}} {{{commitMessageAction}}} {{{commitMessageTopic}}} {{{commitMessageExtra}}} {{{commitMessageSuffix}}}',
+    deprecationMsg:
+      'Direct editing of commitMessage is now deprecated. Please edit commitMessage subcomponents instead.',
```

Comment:
```suggestion
      'Direct editing of the commitMessage is deprecated. Please edit commitMessage subcomponents instead.',
```

The message should probably list the subcomponents, or link to a docs entry that explains the correct procedure.

Reply:
https://docs.renovatebot.com/configuration-options/#commitmessage

There is already a depreciation warning for `commitMessage`, which lists the subcomponents. Maybe we can shorten the `depreciationMsg` instead or there will be a double warning for the same reason. What do you suggest will be a better way to handle this?

Reply:
remove the existing warning in favor of this.

maybe we should add an optional deprecated related options field where we list alternatives.